### ISSUE TITLE
Added transactions like in Redis

### ISSUE THREAD

**Issue by gvsafronov** [tidwall/tile38#640]
**Describe the solution you'd like**
I suggest adding transaction support to the Tile38, such in Redis.
Because in my mind during the execution of instructions in Tile38, multiple consecutively executed instructions were interfered, interrupted, and jumped into the queue.

**Comment by tidwall** [comment]
Do you mean adding `MULTI` command?

**Comment by dibrinsofor** [comment]
Not quite, I think @gvsafronov is talking about `Transactions`. Redis bundles `MULTI`, `EXEC` and `WATCH` into `Transactions` to handle requests sequentially and provide some additional level of atomicity

**Comment by gvsafronov** [comment]
Josh, I mean ACID transactions.
